### PR TITLE
Add guide for log collection

### PR DESCRIPTION
**Pull request by benjaminhuo**
Signed-off-by: Benjamin <benjamin@yunify.com>

### CODE REVIEW

File: content/en/docs/cluster-administration/cluster-settings/log-collections/add-fluentd-as-receiver.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluentd-config
+  namespace: default
+data:
+  fluent.conf: |-
+    # Receive logs sending from Fluent Bit on port 24224
```
Comment (Sherlock113) [inline content/en/docs/cluster-administration/cluster-settings/log-collections/add-fluentd-as-receiver.md:42]: Receive logs sending from Fluent Bit on port 24224
->
Receive logs sent from Fluent Bit on port 24224

File: content/en/docs/cluster-administration/cluster-settings/log-collections/add-fluentd-as-receiver.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+2. Choose ***Fluentd*** and fill in the Fluentd service address and port like below:
+
+![Add Fluentd](/images/docs/cluster-administration/cluster-settings/log-collections/add-fluentd.png)
+
+3. Fluentd appears in the receiver list of ***Log Collections*** UI and its status shows ***Collecting***.
+
+![Receiver List](/images/docs/cluster-administration/cluster-settings/log-collections/receiver-list.png)
+
+
+4. Verify Fluentd is receiving logs sent from Fluent Bit:
```
Comment (Sherlock113) [inline content/en/docs/cluster-administration/cluster-settings/log-collections/add-fluentd-as-receiver.md:144]: Verify Fluentd is receiving logs sent from Fluent Bit:
->
Verify whether Fluentd is receiving logs sent from Fluent Bit:

File: content/en/docs/cluster-administration/cluster-settings/log-collections/add-fluentd-as-receiver.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+This doc will demonstrate:
+
+- How to deploy Fluentd as deployment and create corresponding service and configmap.
+- How to add Fluentd as a log receiver to receive logs sent from Fluent Bit and then output to stdout.
+- How to verify if Fluentd receives logs successfully.
+
+## Prerequisite
+
+To configure log collection, you should use an account of ***platform-admin*** role.
+
```
Comment (Sherlock113) [inline content/en/docs/cluster-administration/cluster-settings/log-collections/add-fluentd-as-receiver.md:19]: Add the following content under this part as well:

You need to enable Logging, Events or Auditing Logs by following [Enable Pluggable Components](https://kubesphere.io/docs/pluggable-components/). Logging is enabled in this example for demonstration.

**Review by Sherlock113 — COMMENTED** [review]
The general process looks clear to me.

Please have a look at my comments and make changes. Thanks.

**Comment by benjaminhuo** [comment]
All comments resolved

**Comment by Sherlock113** [comment]
/lgtm
@FeynmanZhou @huanggze The pr can be merged if you do not have other questions.

**Pull request merged**